tests/jeos/image_info.pm

```diff
@@ -177,8 +177,9 @@ sub run {
     }
 
     # Get the size of different file types
-    unless (is_openstack) {
-        my $btrfs_summary = script_output('btrfs filesystem df --mbytes --si /');
+    # This step applies to BTRFS images. For simplicity, others images will be skipped.
+    my $btrfs_summary = script_output('btrfs filesystem df --mbytes --si /', proceed_on_failure => 1);
+    if ($btrfs_summary !~ /ERROR/) {
```

Comment:
Not sure whether `btrfsprogs` are installed on OpenStack. If the package is not there, you will get `command not found` instead of `ERROR` ( that comes from `btrfsprogs`, right? )

Reply:
You are super right https://openqa.suse.de/tests/8793555#step/image_info/8

Reply:
Amended